contracts/external/interfaces/ICurve.sol

```diff
@@ -0,0 +1,44 @@
+/*
+
+    Copyright 2020 dYdX Trading Inc.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+*/
+
+pragma solidity 0.5.7;
+pragma experimental ABIEncoderV2;
+
+
+/**
+ * @title ICurve
+ * @author dYdX
+ *
+ * Partial interface for a Curve contract.
+ */
+interface ICurve {
+
+    function fee()
+        external
+        view
+        returns (uint256);
+
+    function get_dy_underlying(
```

Comment:
do you prefer `get_dy_underlying` over `get_dy` as `get_dy_underlying` works on more types of Curve pools? Are there any gas implications to that?

Reply:
From the source code I thought `get_dy_underlying` was going to be cheaper, but looks like that's not the case. `55037` vs `55572`. So I'll use `get_dy` instead